### ISSUE TITLE
Menu item items

### ISSUE THREAD

**Issue by Scotttie** [adventistmedia/simple_church#120]
- external/internal link 'Target' is tech descriptions. Can we dumb it down for simple users, like radio buttons:

Open link in:
>Same window
>New window 

- and should the 'new window' be default, knowing it's an external link.

- URL, has the hint 'https://' in the field and disappears when typing, so, part of me thinks I have to type in the https://, or do I??.
Would it be slightly better to have a full address in there, like 'https://www.website.com'
whatever you think.

- being able to edit an internal link's URL in the menu, If I edit, does that save as teh new SLUG for that page? If not, it should not be possible to edit.

- the top menu item does not need a 'sub menu' arrow.

**Comment by danlewis** [comment]
Changes implemented on all points.
